server/datastore/mysql/software.go

```diff
@@ -436,13 +437,18 @@ func getOrGenerateSoftwareIdDB(ctx context.Context, tx sqlx.ExtContext, s fleet.
 	}
 
 	_, err := tx.ExecContext(ctx,
-		"INSERT INTO software "+
-			"(name, version, source, `release`, vendor, arch, bundle_identifier, extension_id, browser) "+
-			"VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)",
+		fmt.Sprintf("INSERT INTO software "+
+			"(name, version, source, `release`, vendor, arch, bundle_identifier, extension_id, browser, checksum) "+
+			"VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, %s)", softwareChecksumComputedColumn("")),
```

Comment:
@roperzh In an ideal world, this change would get load-tested. I don't think we have other load tests planned, unless you run another one to verify the optimizations done following the software titles load test? If so it would be great to include that change in the tested codebase, to check that osquery ingestion performance/load is unchanged.

Reply:
sounds good, will try to squeeze another load test to validate the software fixes as well

Reply:
Endpoint ops is planning to do load test for live query stats changes.